src/Microsoft.Sbom.Common/Config/Configuration.cs

```diff
@@ -339,4 +340,13 @@ public ConfigurationSetting<string> SbomDir
         get => sbomDir.Value;
         set => sbomDir.Value = value;
     }
+
+    /// <inheritdoc cref="IConfiguration.ComplianceStandard" />
+    public ConfigurationSetting<ComplianceStandard> ComplianceStandard
+    {
+        get => complianceStandard.Value;
+        set => complianceStandard.Value = value;
+    }
+
+    ConfigurationSetting<string> IConfiguration.ComplianceStandard { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
When will this get implemented?

Reply:
Seems like I accidentally left this line in, updated the getters and setters so this should make more sense now.